test/e2e/runner/cli_provisioner.go

```diff
@@ -135,13 +135,6 @@ func (cli *CLIProvisioner) provision() error {
 		return fmt.Errorf("Error while trying to create deployment:%s", err)
 	}
 
-	if cli.CreateVNET {
```

Comment:
This is in the ginkgo E2E runner. We're converting the custom VNET attachment over to Azure CNI.